package.json

```diff
@@ -140,6 +141,7 @@
     "jest-axe": "^6.0.0",
     "jest-environment-jsdom": "^29.0.2",
     "jest-fetch-mock": "^3.0.3",
+    "jest-snapshot-serializer-ansi": "^1.0.0",
```

Comment:
Does not look like an actively maintained package, can it be dropped?

Reply:
It doesn't need to be. All it does is strip ANSI (region locale) string blocks. This is needed if you have different locale settings in your computer, so that the snapshots keep matching doesn't matter where.